README.md

```diff
@@ -509,8 +510,9 @@ const User = ({ name, description, theme: { marginLeft, height } }) => (
 
 ## It's easier to remember an interface if it resembles one you've seen before
 
-Modelling your API on something similar and well-known makes it faster for
-other developers to understand.
+Modelling your API on something similar and well-known makes it faster for other
+developers to understand. In general, it's most effective to spread all
+additional props. More on [Pass props](#passprops).
```

Comment:
I believe Github automatically id-ifies headings, so this link should be able to point to
```suggestion
additional props. More on [Pass props](#pass-props).
```